content/projects/microbial/index.md

```diff
@@ -0,0 +1,137 @@
+
+Want to include your workflow here? Let us know!
+
+## A dedicated interface
+
+A dedicated interface with tools, workflows for microbial data analyses can be accessed via [microgalaxy.usegalaxy.eu](https://microgalaxy.usegalaxy.eu/)
+
+## Training material and events
+
+The Galaxy community organizes regularly training events. You can check the [event pages here](/events/) to get the last ones.
+
+We would like to highlight few events related to microbial data analysis:
+- [GTN Smörgåsbord 2023](https://gallantries.github.io/video-library/events/smorgasbord3/): a free, online, self-paced, choose-your-own-adventure style, 1-week event in May 2023
+
+  There are several tracks that might be interesting for you:
+  - [Meta-omics](https://gallantries.github.io/video-library/modules/meta-omics)
+  - [Microbial Analysis](https://gallantries.github.io/video-library/modules/microbial)
+
+
+# How is Galaxy used for microbial data analysis?
+
+Below is a list of projects involving members of this community:
+
+| Project | Description | Techniques | Sequencing | Analyses | People involved | Funding | Status |
+|---------|-------------|------------|------------|----------|-----------------|---------|--------|
+| [IRIDA](https://irida.ca/) | The Integrated Rapid Infectious Disease Analysis | | | | Aaron Petkau
+Foodborne pathogen detection | | Metagenomics | Long reads | | Bérénice Batut, Engy Nasr | EOSC-Life for 2022 | Ongoing
+| [BeerDEcoded](https://streetscience.community/projects/beerdecoded/) | General public education project | Metagenomics | Long reads |   | Bérénice Batut, [Street Science Community](https://streetscience.community/) | Mozilla, de.NBI | Ongoing
+| [DNAnalyzer](http://streetscience.community/DNAnalyzer/) | Online & interactive game on DNA data analysis | Metagenomics | Long reads | | Bérénice Batut, [Street Science Community](https://streetscience.community/) | University of Freiburg | Ongoing
+| Cloud data | | Metagenomics, metatranscriptomics | Short reads | Taxonomy profiling, Functional profiling, Assembly, MAGs builiding | Bérénice Batut, Engy Nasr | | Ongoing
+| ASaiM | | Metagenomics, metatranscriptomics | Short reads | Taxonomy profiling, Functional profiling | Bérénice Batut, Saskia Hilteman, Pratik Jagtap, Subina Mehta, etc | | Finished?
+| [Mycobacterium tuberculosis NGS made easy: data](https://gallantries.github.io/video-library/modules/mtb-analysis) | Series of modules containing recorded videos by experts and tutorials | | Illumina (wanting to expand to nanopore) | Mapping & Variant calling, Molecular epidemiology of TB using NGS, Phylogenetics for studying TB evolution and epidemiology | Daniela Brites, Peter van Heusden, Galo A. Goig, Christoph Stritt | Initially funded trough TB projects of capacity building; currently no funding | Ready to be used  but also working on improvements
+
+
+## Projects / Showcases
```

Comment:
Should this not be before the last table. Or should this become a specific topic ?

Reply:
Yes. The title should be before "Below"